### PR TITLE
Add secure update methods

### PR DESCRIPTION
**Pull request by mstemm**
@03cranec do you want to double-check the terminology I used for the split? check the first commit for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 [![Current version on PyPI](http://img.shields.io/pypi/v/sdcclient.svg)](https://pypi.python.org/pypi/sdcclient)
 
-A Python client API for Sysdig Cloud.
+A Python client API for Sysdig Monitor/Sysdig Secure.
 
-This module is a wrapper around the Sysdig Cloud API, which is documented [here](http://support.sysdigcloud.com/hc/en-us/articles/205233166-The-Sysdig-Cloud-API-Specification). It exposes most of the sysdig REST API functionality as an easy to use and easy to install Python interface. The repository includes a rich set of examples (in the [examples](examples/) subdir) that quickly address several use cases.
+This module is a wrapper around the Sysdig Monitor/Sysdig Secure APIs, which are documented [here](http://support.sysdigcloud.com/hc/en-us/articles/205233166-The-Sysdig-Cloud-API-Specification) and [here](xxx). It exposes most of the sysdig REST API functionality as an easy to use and easy to install Python interface. The repository includes a rich set of examples (in the [examples](examples/) subdir) that quickly address several use cases.
```
Comment (philrz) [inline README.md]: Do you have a link to put in for that "xxx"?

File: README.md [hunk 2]
```diff
@@ -20,36 +20,38 @@ Installation
...
 #
-sdclient = SdcClient(sdc_token)
+client = SdMonClient(api_token)
 ```
 
+For backwards compatibility purposes, a third class `SdcClient` is exported which is an alias of `SdMonClient`.
+
 Once instantiated, all the methods documented below can be called on the object.
 
 ####Return Values
```
Comment (philrz) [inline README.md]: This wasn't your doing, but as long as you're in the code, might as well fix the fact that this "Return Values" bit doesn't render into a Markdown header.

**Review by philrz — COMMENTED** [review]
tl;dr - Docs.

Background on docs: The current docs are generated via Sphinx using docstrings. What this gives us is that if you go to the `docs/` directory and type `make html`, it'll create docs with hyperlinks and all the trimmings (also options for PDF. etc.) And since we have that, it's also rigged up so that every time we commit something in GitHub, it triggers the recompile of the HTML docs and publishing them at http://python-sdc-client.readthedocs.io/en/latest/.

This leads to two things:

1. Ideally, the newly-added methods should have such docstrings. I know not every legacy method in Sysdig Monitor has them, but when I did the Sphinx conversation I at least brought all the semi-broken ones up to date. So it'd be good to be thorough going forward.

2. More importantly, splitting out the client into "common" and separate ones for Monitor/Secure has broken the current compilation of the docs. Unfortunately, while I got Sphinx working and was happy with it, I confess I am not a Sphinx *expert*. Its own docs confuse the hell out of me, and surely part of that is because while I can get by in Python, I am by no means an expert in Python in general.

Now, with a quick hack, it looks like the following diff at least gets the docs generating again:

```diff --git a/doc/index.rst b/doc/index.rst
index 9e27bb9..acdbdba 100644
--- a/doc/index.rst
+++ b/doc/index.rst
@@ -16,6 +16,11 @@ This page documents the functions available in the `Python Script Library <https
 Function List
 =============
 .. py:module:: sdcclient
-.. autoclass:: SdcClient
+.. autoclass:: SdMonClient
    :members:
+   :inherited-members:
+   :undoc-members:
+.. autoclass:: SdSecureClient
+   :members:
+   :inherited-members:
    :undoc-members:
```
What's maybe not great about what comes out of this is that it basically documents Monitor and Secure as separate whole animals as if the "common" methods they inherit belong to each, i.e. the page repeats the docstring for `get_user_info()`, etc. I did some quick searches to see how people deal with this problem, but nothing stood out. Maybe we just live with it. If you have some cycles to look deeper into it, that would be great.

File: README.md (hunk 1 above)
Comment (mstemm) [reply README.md]: Now that I know about readthedocs, I'll drop the second link and just point everything to the one page.

File: README.md (hunk 2 above)
Comment (mstemm) [reply README.md]: Fixed.

**Comment by mstemm** [comment]
Ok, added sphinx docs and fixed the README. Wanna take a look again?

**Comment by philrz** [comment]
Mmkay. Docs look good and automated tests are all passing, so, LGTM. :+1:
